src/activity_handlers.cpp

```diff
@@ -2769,6 +2775,146 @@ void activity_handlers::try_sleep_do_turn( player_activity *act, player *p )
     }
 }
 
+void activity_handlers::operation_do_turn( player_activity *act, player *p )
+{
+    /**
+    - values[0]: Difficulty
+    - values[1]: success
+    - values[2]: max_power_level
+    - values[3]: pl_skill
+    - values[4: ]: occupied body_parts
```

Comment:
```suggestion
    - values[4]: occupied body_parts
```

Reply:
So I know it's not the right synthaxe but I mean starting at 4 and above